profile/Makefile

```diff
@@ -0,0 +1,170 @@
+#===============================================================================
+# User Options
+#===============================================================================
+
+COMPILER    = gnu
+MPI         = no
+OPENMP      = yes
+OPTIMIZE    = yes
+DEBUG       = no
+PROFILE     = no
+PAPI        = no
+BENCHMARK   = no
+PRECISION   = single
+
+#===============================================================================
+# Program name & source code list
+#===============================================================================
+
+case = c5g7-cmfd.cpp
+
+source = \
+../src/Cell.cpp \
+../src/Cmfd.cpp \
+../src/CPUSolver.cpp \
+../src/ExpEvaluator.cpp \
+../src/Geometry.cpp \
+../src/LocalCoords.cpp \
+../src/log.cpp \
+../src/Material.cpp \
+../src/Point.cpp \
+../src/PolarQuad.cpp \
+../src/Solver.cpp \
+../src/Surface.cpp \
+../src/Timer.cpp \
+../src/Track.cpp \
+../src/TrackGenerator.cpp \
+../src/Universe.cpp
+
+cases = \
+one-directional-gradient.cpp \
+two-directional-gradient.cpp \
+homogeneous-one-group.cpp \
+c5g7.cpp \
+c5g7-cmfd.cpp
+
+obj = $(source:.cpp=.o)
+headers = $(source:.cpp=.h)
+obj += $(case:.cpp=.o)
+
+program = $(case:.cpp=)
+
+#===============================================================================
+# Sets Flags
+#===============================================================================
+
+# Regular gcc Compiler
+ifeq ($(COMPILER),gnu)
+  CC = g++
+endif
+
+# intel Compiler
+ifeq ($(COMPILER),intel)
+  CC = icc
```

Comment:
If the Intel compiler is used, you might also append "VectorizedSolver.cpp" to the list of sources. I'd actually be very interested to see that guy profiled to determine wth is wrong with it.